index.ts

```diff
@@ -0,0 +1,186 @@
+// https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md#run-lighthouse-multiple-times
+import { spawnSync } from 'child_process';
+import { computeMedianRun } from 'lighthouse/lighthouse-core/lib/median-run.js';
+import psi from 'psi';
+// https://stackoverflow.com/a/62499498/9931154
+import { createRequire } from 'module';
+import { type Options } from './options.js';
+const metaRequire = createRequire(import.meta.url);
+const lighthouseCli = metaRequire.resolve('lighthouse/lighthouse-cli');
```

Comment:
Do you know if this `createRequire` is still needed? I have no idea why `import.meta` was needed, but I know `import.meta` is a super annoying old JavaScript thing I don't understand.

Reply:
Unsure, but it seems to work as is and I didn't understand it either so I didn't mess with it initially. I can try updating this to a more modern import and see if that works though.

Reply:
Okay, it's fine to leave it as is. `import.meta` is so frustrating to work with.